.github/mention-to-slack.yml

```diff
@@ -0,0 +1,15 @@
+# For Github User
+# github_username: "slack_member_id"
+# For Github Team
+# github_teamname: "slack_member_id"
```

Comment:
Casual switch to single quotes in the comments to avoid future copy-and-pastes making the formatting check fail:

```suggestion
# For GitHub user:
# github_username: "slack_member_id"
# For GitHub team:
# github_teamname: "slack_member_id"
```